net/strongswan/files/swanctl.init

```diff
@@ -359,8 +357,6 @@ config_child() {
 		;;
 	esac
 
-	[ -n "$local_nat" ] && local_ts="$local_nat"
```

Comment:
People sometimes have to deal with NATting RFC-1918 collisions in their address space for reasons beyond their control, or for legacy reasons, etc. They don't always have the option to do otherwise.

Reply:
Is the presence/definition of local_nat in the UCI config having any influence on anything but the swanctl.init script? What happens if we just remove it?

Reply:
Why remove it?  I thought we were trying to expose as many underlying swanctl controls as possible for completeness...